sepa/src/main/res/layout/sepa_view.xml

```diff
@@ -22,8 +20,9 @@
         android:layout_height="wrap_content">
 
         <com.adyen.checkout.components.ui.view.AdyenTextInputEditText
-            tools:ignore="RequiredSize"
-            style="@style/AdyenCheckout.Sepa.HolderNameInput" />
+            android:id="@+id/editText_holderName"
+            style="@style/AdyenCheckout.Sepa.HolderNameInput"
+            tools:ignore="RequiredSize" />
```

Comment:
Why don't we specify the size? This could cause some issues

Reply:
We have this attribute in 58 places 😅
I don't know what's the thought behind it, maybe for merchants to be able to override width and height using `styles.xml`? 🤔
In all cases I think it needs its own PR if we were to remove it.

Reply:
Right, let's look at it some other time 👍